dotfiles/bashrc

```diff
@@ -88,6 +88,9 @@ function prod() {
   $* --app $(basename $(pwd))-production
 }
 
+yolo() {
+  brew update && brew outdated && brew upgrade && brew postgresql-upgrade-database || brew cleanup && brew prune
```

Comment:
is there any point in listing outdated deps in the middle of this? afaik `brew outdated` doesn't actually make any changes. And I think the output will get lost in the middle of all the other commands

Reply:
I added it because I wanted a list of everything getting updated in one spot.